### PR TITLE
Commented out xref links fix #1

### PR DESCRIPTION
**Pull request by florinciubotariu**
Relates to https://github.com/dotnet/docs/issues/3682.

### CODE REVIEW

File: docs/framework/wcf/how-to-host-and-run-a-basic-wcf-service.md [hunk 1]
```diff
@@ -145,7 +145,7 @@ This is the third of six tasks required to create a Windows Communication Founda
...
...xref:System.ServiceModel.WSHttpBinding> which is a built-in binding that is used for connecting to endpoints that conform to the WS-* specifications. For more information about WCF bindings, see [WCF Bindings Overview](../../../docs/framework/wcf/bindings-overview.md). The address is appended to the base address to identify the endpoint. The address specified in this code is "CalculatorService" so the fully qualified address for the endpoint is `"http://localhost:8000/GettingStarted/CalculatorService"` Adding a service endpoint is optional when using .NET Framework 4.0 or later. In these versions, if no endpoints are added in code or configuration, WCF adds one default endpoint for each combination of base address and contract implemented by the service. For more information about default endpoints see [Specifying an Endpoint Address](../../../docs/framework/wcf/specifying-an-endpoint-address.md). For more information about default endpoints, bindings, and behaviors, see [Simplified Configuration](../../../docs/framework/wcf/simplified-configuration.md) and [Simplified Configuration for WCF Services](../../../docs/framework/wcf/samples/simplified-configuration-for-wcf-services.md).
```
Comment (mairaw) [inline docs/framework/wcf/how-to-host-and-run-a-basic-wcf-service.md]: System.ServiceModel.ServiceEndpoint -> System.ServiceModel.Description.ServiceEndpoint

File: docs/framework/windows-workflow-foundation/samples/using-the-invokemethod-activity.md [hunk 2]
```diff
@@ -4,9 +4,9 @@ ms.date: "03/30/2017"
 ms.assetid: b6643550-d043-4ac7-8069-9c55ebbb4233
 ---
 # Using the InvokeMethod Activity
-This sample demonstrates how to use the <!--zz <xref:System.Activities.Statements.InvokeMethod%601> -->[<code>System.Activities.Statements.InvokeMethod\`1</code>](https://msdn.microsoft.com/library/dd647677.aspx) activity to invoke public methods in public classes. The <!--zz <xref:System.Activities.Statements.InvokeMethod%601> -->[<code>System.Activities.Statements.InvokeMethod\`1</code>](https://msdn.microsoft.com/library/dd647677.aspx) activity allows a workflow to call methods against objects, pass in parameters, get the return value, specify types for generic methods, and specify whether the method is synchronous or asynchronous. 
+This sample demonstrates how to use the <xref:System.Activities.Statements.InvokeMethod%601> activity to invoke public methods in public classes. The <xref:System.Activities.Statements.InvokeMethod%601> activity allows a workflow to call methods against objects, pass in parameters, get the return value, specify types for generic methods, and specify whether the method is synchronous or asynchronous.
```
Comment (mairaw) [inline docs/framework/windows-workflow-foundation/samples/using-the-invokemethod-activity.md]: This a tricky change and probably will be more involved. This topic should probably be deleted. The sample doesn't exist and System.Activities.Statements.InvokeMethod<T> was removed at some point apparently. My suggestion would be to delete this file and add an entry to https://docs.microsoft.com/en-us/dotnet/framework/windows-workflow-foundation/samples/built-in-activities in the https://github.com/dotnet/docs/blob/master/.openpublishing.redirection.json file. Let us know if you need any help with this,

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks for trying to tackle these @florinciubotariu. Please take a look at my comments.

**Comment by florinciubotariu** [comment]
@mairaw That's new for me :). Hope it is ok!

**Review by mairaw — APPROVED** [review]
You did great @florinciubotariu! Thank you so much!

**Pull request merged**